### DISCUSSION TITLE
Rocket 0.5 upgrade

### DISCUSSION THREAD

**Discussion by nazar554** [dani-garcia/vaultwarden#4082] [Ideas]
Rocket 0.5 just got released, might be the time to switch from -rc version

https://rocket.rs/v0.5/news/2023-11-17-version-0.5/

**Comment by BlackDex** [comment]
We just updated to rc4 last weekend, it probably doesn't have much changes so, testing is probably already kinda up-to-date.

But thanks for mentioning!

**Comment by BlackDex** [comment]
I quickly checked the differences between rc4 and v0.5 final, and the only changes are readme files or comments, nothing in the code it self. So or testing tagged images are already up to date.
